samples/api/dynamic_uniform_buffers/dynamic_uniform_buffers.cpp

```diff
@@ -272,14 +272,18 @@ void DynamicUniformBuffers::setup_descriptor_set()
 
 	VK_CHECK(vkAllocateDescriptorSets(get_device().get_handle(), &alloc_info, &descriptor_set));
 
-	VkDescriptorBufferInfo            view_buffer_descriptor    = create_descriptor(*uniform_buffers.view);
-	VkDescriptorBufferInfo            dynamic_buffer_descriptor = create_descriptor(*uniform_buffers.dynamic);
-	std::vector<VkWriteDescriptorSet> write_descriptor_sets     = {
-        // Binding 0 : Projection/View matrix uniform buffer
-        vkb::initializers::write_descriptor_set(descriptor_set, VK_DESCRIPTOR_TYPE_UNIFORM_BUFFER, 0, &view_buffer_descriptor),
-        // Binding 1 : Instance matrix as dynamic uniform buffer
-        vkb::initializers::write_descriptor_set(descriptor_set, VK_DESCRIPTOR_TYPE_UNIFORM_BUFFER_DYNAMIC, 1, &dynamic_buffer_descriptor),
-    };
+	VkDescriptorBufferInfo view_buffer_descriptor    = create_descriptor(*uniform_buffers.view);
+	VkDescriptorBufferInfo dynamic_buffer_descriptor = create_descriptor(*uniform_buffers.dynamic);
+
+	// Override the default VK_WHOLE_SIZE range for the descriptor with the actual dynamic alignment
+	dynamic_buffer_descriptor.range = dynamic_alignment;
```

Comment:
Why not just
`	VkDescriptorBufferInfo            dynamic_buffer_descriptor = create_descriptor(*uniform_buffers.dynamic, dynamic_alignment);`
?

Reply:
Good point. I missed that constructor. Will update the PR.